.github/dependabot.yml

```diff
@@ -22,3 +22,9 @@ updates:
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 10
+    ignore:
+      # upstream dependencies that need to be updated in sync and with manual control
```

Comment:
[for reviewer] those dependencies will need to be manually updated by using the script or through manual edits (for example the current 2.1.0 release does not contain any dependency on the contrib that we could reuse, so we have to do it manually for now).